firmware/components/driver_framebuffer/driver_framebuffer_matrix.cpp

```diff
@@ -143,7 +144,7 @@ esp_err_t matrix_stack_2d_push(matrix_stack_2d *stack) {
     return ESP_OK;
 }
 
-//returns ESP_ERROR if the stack is already empty
+//returns 1 if the stack is already empty
```

Comment:
shouldn't you use ESP_ERR_INVALID_ARG here? (and return ESP_ERR_INVALID_ARG instead of 1)

Reply:
I could do that, but i don't think it's a problem to keep this.

Reply:
See my comment a few lines earlier; I agree that the `1` is now more in line with the `return 1;` in the code. But without comments it's a bit strange to return 1 instead of an `esp_err_t` constant. I think it would be a good idea to mention the old deprecated ESP_ERROR constant then.

Reply:
Now that i think of it, ESP_ERR_INVALID_ARG is more wrong than this is.
It's not an invalid argument that's causing an error.
This is a pop operation.
It is a stack underflow at most.

Reply:
Hmm, you're right.. I was pretty sure there was an `ESP_ERROR` one day, but I cannot find it. I don't know where the `1` came from, then. I would suggest `// returns 1 (ESP_ERROR?!) ...` or something like that.. But you can also ignore me.. I think fixing the code itself is outside the scope of the merge request.

All `ESP_ERR_*` defines can be found here: https://github.com/espressif/esp-idf/blob/286202caa31b61c2182209f37f8069a0b60fb942/components/esp32/include/esp_err.h